lxd/cgroup/abstraction.go

```diff
@@ -827,6 +827,46 @@ func (cg *CGroup) GetMemoryStats() (map[string]uint64, error) {
 	return out, nil
 }
 
+// GetOOMKills returns the number of oom kills.
+func (cg *CGroup) GetOOMKills() (int64, error) {
+	var (
+		err   error
+		stats string
+	)
+
+	version := cgControllers["memory"]
+
+	switch version {
+	case Unavailable:
```

Comment:
Probably better to use a the default switch case here.

Reply:
Yes

Reply:
done